util/populate_dev_data.py

```diff
@@ -0,0 +1,175 @@
+
+    remote_api_stub.ConfigureRemoteApiForOAuth(
+        args.server_uri,
+        '/_ah/remote_api',
+        secure=args.secure)
+
+    class Token(ndb.Model):
+        Secret = ndb.StringProperty()
+
+    class TestRun(ndb.Model):
+        BrowserName = ndb.StringProperty()
+        BrowserVersion = ndb.StringProperty()
+        OSName = ndb.StringProperty()
+        OSVersion = ndb.StringProperty()
+        Revision = ndb.StringProperty()
+        ResultsURL = ndb.StringProperty()
+        CreatedAt = ndb.DateProperty()
+
+    # Create empty Token 'upload-token'
+    secret = Token(
+        id='upload-token',
+        Secret='')
+    secret.put()
+    logging.info('Added Token \'upload-token\' with empty secret.')
+    
+    # Add some runs.
+    path = '/static/b952881825/%s'
+    test_runs = [
+        TestRun(
+            id='dev-testrun-chrome-63',
+            BrowserName='chrome',
+            BrowserVersion='63.0',
+            OSName='linux',
+            OSVersion='3.16',
+            Revision='b952881825',
+            ResultsURL=path % 'chrome-63.0-linux-summary.json.gz'),
+        TestRun(
+            id='dev-testrun-edge-15',
+            BrowserName='edge',
+            BrowserVersion='15',
+            OSName='windows',
+            OSVersion='10',
+            Revision='b952881825',
+            ResultsURL=path % 'edge-15-windows-10-sauce-summary.json.gz'),
+        TestRun(
+            id='dev-testrun-firefox-57',
+            BrowserName='firefox',
+            BrowserVersion='57.0',
+            OSName='linux',
+            OSVersion='*',
+            Revision='b952881825',
+            ResultsURL=path % 'firefox-57.0-linux-summary.json.gz'),
+        TestRun(
+            id='dev-testrun-safari-10',
+            BrowserName='safari',
+            BrowserVersion='10',
+            OSName='macos',
+            OSVersion='10.12',
+            Revision='b952881825',
+            ResultsURL=path % 'safari-10-macos-10.12-sauce-summary.json.gz'),
+    ]  # type: List[TestRun]
+
+    for test_run in test_runs:
+        test_run.put()
+        logging.info('Added TestRun %s' % test_run.key.id())
+
+    # Also whatever the latest TestRun are.
+    logging.debug('Added TestRun %s' % test_run.key.id())
+    copier = ProdRunCopier(logging.getLogger())
+    copier.copy_prod_run('latest')
+
+
+# Create an ArgumentParser for the flags we'll expect.
+def parse_flags():  # type: () -> argparse.Namespace
+    # Re-use the docs above as the --help output.
+    parser = argparse.ArgumentParser(description=inspect.cleandoc(__doc__))
+    parser.add_argument(
+        '--log',
+        type=str,
+        default='INFO',
+        help='Log level to output')
+    parser.add_argument(
+        '--sdk-root',
+        type=str,
+        dest='sdk_root',
+        default='',
+        help='Root path to the App Engine SDK installation, if it\'s not'
+             'already in your PYTHONPATH. You can download the SDK from '
```

Comment:
Sorry, I just spotted another bit here: need a space after "not", or before "already", otherwise is prints out as "notalready"